### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ehulinian**
- [DEMO LINK](https://Ehulinian.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Ehulinian.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,12 +2,53 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__review-count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price-container">
+        <div class="card__price-label">Price:</div>
+        <div class="card__price">$2,199</div>
```
Comment (etojeDenys) [inline src/index.html:42]: better to use some semantic tags here for text

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,123 @@
...
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(/src/images/star-active.svg);
```
Comment (etojeDenys) [inline src/styles/index.scss:122]: move to another file as a separate block

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,123 @@
...
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--text-accent-color);
+}
```
Comment (etojeDenys) [inline src/styles/index.scss:38]: use sass nesting

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,123 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
```
Comment (etojeDenys) [inline src/styles/index.scss:4]: use sass vars instead of CSS

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+
+  .card__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
```
Comment (l4st1m0za) [inline src/styles/blocks/card.scss:29]:
```suggestion
.card {
  box-sizing: border-box;
  width: 200px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
  padding: 32px 16px 16px;
  background: #fff;

  &__image {
    display: block;
    margin: 0 auto 40px;
    width: 160px;
    height: 134px;
  }
```

Use SASS nesting like this in **ALL THE CASES**

**Review by l4st1m0za — APPROVED** [review]
Good job, see the comment

**Pull request closed without merging**